checkbox-provider/lib/qat_tools-1.0.0/src/qat_tools/qatctl.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+from qat_tools.qatlib import *
+
+import argparse
+from prettytable import PrettyTable
+
+VERSION = '1.0.0'
+
+def list_dev(args, qat_manager):
+  qat_manager.list_devices(args.short)
+
+def status_dev(args, qat_manager):
+  if args.vfio:
+    qat_manager.print_vfio()
+  else:
+    qat_manager.print_cfg()
+
+# Check arguments and call requested function
+def qatctl(opts, p):
+  global VERSION
+  qat_manager = QatDevManager(opts.devices)
+
+  if opts.set_state:
+    print(f'Set device state : {opts.set_state}')
+    qat_manager.set_state(opts.set_state)
+    return
+
+  if opts.get_state:
+    qat_manager.get_state()
+    return
+
+  if opts.set_service:
+    print(f'Set device service : {opts.set_service}')
+    qat_manager.set_service(opts.set_service)
+    print(f'Please restart qat service to update the config')
```

Comment:
What do you think about using the python `logging` module instead of print statements?

Reply:
`logging` can be nice but for now, the tools are supposed to print information to users (stdout), they are not supposed to do logging,